pkg/security/secl/model/endianness/byteorder.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-package model
```

Comment:
If you wanted to, you can use `pkg/util/native.Endian` for this. We actually take advantage of the determining the byte order at build time.

Reply:
Interesting 🤔 I remember reading somewhere that ARM can in theory support both little and big endian ... Have you encountered this somewhere ?

Reply:
Build tags 🎉 Thanks Bryce !

Reply:
Updated for `cws-injector` ! I'll update the rest of CWS in a separate PR.